Tasks/Maven/maven.ps1

```diff
@@ -5,19 +5,31 @@
     [string]$publishJUnitResults,   
     [string]$testResultsFiles, 
     [string]$jdkVersion,
-    [string]$jdkArchitecture
-)
-
-Write-Verbose 'Entering Maven.ps1'
-Write-Verbose "mavenPOMFile = $mavenPOMFile"
-Write-Verbose "options = $options"
-Write-Verbose "goals = $goals"
-Write-Verbose "publishJUnitResults = $publishJUnitResults"
-Write-Verbose "testResultsFiles = $testResultsFiles"
-Write-Verbose "jdkVersion = $jdkVersion"
-Write-Verbose "jdkArchitecture = $jdkArchitecture"
-
-#Verify Maven POM file is specified
+    [string]$jdkArchitecture,
+    [string]$sqAnalysisEnabled, 
+    [string]$sqConnectedServiceName, 
+    [string]$sqDbDetailsRequired,
+    [string]$sqDbUrl,
+	[string]$sqDbUsername,
+	[string]$sqDbPassword)
+
+
+Write-Verbose -Verbose 'Entering Maven.ps1'
+Write-Verbose -Verbose "mavenPOMFile = $mavenPOMFile"
+Write-Verbose -Verbose "options = $options"
+Write-Verbose -Verbose "goals = $goals"
+Write-Verbose -Verbose "publishJUnitResults = $publishJUnitResults"
+Write-Verbose -Verbose "testResultsFiles = $testResultsFiles"
+Write-Verbose -Verbose "jdkVersion = $jdkVersion"
+Write-Verbose -Verbose "jdkArchitecture = $jdkArchitecture"
+
+Write-Verbose -Verbose "sqAnalysisEnabled = $sqAnalysisEnabled"
```

Comment:
You shouldn't pass in the -Verbose parameter to Write-Verbose. Doing that will cause the output to be always shown without respecting the verbose settings on the build. To see the output during the build set system.debug variable on the build definition to true.